asyncscope.md

```diff
@@ -2358,31 +2358,147 @@ throwing.
 
 ## `execution::nest`
 
-Add the following as a new subsection immediately after __[exec.stopped.as.error]__:
+Add the following as a new subsection immediately after [exec.stopped.err]{.sref}:
 
 ::: add
 __`std::execution::nest` [exec.nest]__
 
 [1]{.pnum} `nest` tries to associate a sender with an async scope such that the scope can track the lifetime of any
 async operations created with the sender.
 
-[2]{.pnum} The name `nest` denotes a customization point object. For subexpressions `sndr` and `token`, let `Sndr` be
-`decltype((sndr))` and let `Token` be `decltype((token))`. If `sender<Sndr>` or `async_scope_token<Sender>` is false,
-the expression `nest(sndr, token)` is ill-formed.
+[2]{.pnum} Let _`nest-data`_ be the following exposition-only class template:
+```cpp
+namespace std::execution {
 
-[3]{.pnum} Otherwise, the expression `nest(sndr, token)` is expression-equivalent to:
+template <async_scope_token Token, sender Sender>
+struct @_nest-data_@ {
+  using @_wrap-sender_@ = decay_t<decltype(declval<Token&>().wrap(declval<Sender>()))>;
+  using @_association_@ = decltype(declval<Token&>().try_associate());
```

Comment:
These should use the expo-only _`association-from`_ and _`wrapped-sender-from`_.